application/predictionengine/src/predictionengine/dataset.py

```
@@ -174,11 +174,15 @@ def batches(self) -> Generator[Tuple[Tensor, Tensor, Tensor], None, None]:
 
             tickers = batch_data[: self.batch_size, 0]
 
+            batch_tensors = [
+                batch_data[i : i + self.sequence_length, 1:]
+                for i in range(self.batch_size)
+            ]
+
+            # TODO: add check for empty batch tensors or size equal to one
```

Pylint (3.3.7)

[warning] TODO: add check for empty batch tensors or size equal to one

(W0511)